week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -31,11 +34,11 @@ Once you have completed your object run the following and see if your answer
 matches the expected result at the bottom :)
 */
 
-bottle.fill();
-bottle.drink();
-bottle.drink();
-bottle.drink();
-if (!bottle.empty()) {
-  console.log(`bottles volume = ${bottle.volume}`);
+bottle.fill ();
+bottle.drink ();
+bottle.drink ();
+bottle.drink ();
+if (!bottle.empty ()) {
+  console.log (`bottles volume = ${bottle.volume}`);
 }
-console.log("Above volume should be: 70");
+console.log ('Above volume should be: 70');

```

Comment:
you like to change " to ' i think it makes no big difference it used when you have both in one sentence i think let me check

Reply:
it is the visual studio and prettier is changing it automatically Constantine